tests/doctrine/DoctrineCleanInsert1Test.php

```diff
@@ -182,11 +182,11 @@ public function testShowDoctrineReadFailiureUntilCommitted(){
     $this->em->persist($site);
     // After persist, Doctrine querying does not return our site until we flush, hence we get 0
     // returned from our repository.
-    $this->assertEquals(0, count($this->em->getRepository('Site')->findOneBy(array('shortName' => 'site' . $n))));
+    $this->assertEquals(0, count($this->em->getRepository('Site')->findBy(array('shortName' => 'site' . $n))));
```

Comment:
What was the error here? It [looks like](https://www.doctrine-project.org/projects/doctrine-orm/en/2.11/tutorials/getting-started.html) the findOneBy methond still exists in Doctrine 2.11...

Reply:
![image](https://user-images.githubusercontent.com/20706670/167655073-faf217a3-c915-44a6-a93f-e6c686ef3860.png)
Something got tightened up.